recipes/epics-base/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "3.14.12.8" %}
+{% set sha256 = "b11eab2fdd01c60219be48954631f22230107380961e0df7c356f7cc8bb5f24e" %}
+
+package:
+  name: epics-base
+  version: {{ version }}
+
+source:
+  fn: baseR{{version}}.tar.gz
+  url: https://epics.anl.gov/download/base/baseR{{version}}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - config_site.diff  # [not win]
+    - config_common.diff  # [not win]
+    - rules_build.diff
+    - config_readline.diff  # [linux]
+
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc14  # [win and (py35 or py36)]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - python  # [win]
+    - make  # [not win]
+    - m2w64-make  # [win]
+    - perl  # [win]
+    - patch  # [win]
```

Comment:
are you using the lib or headers of ``patch``? If you put this as dependency just because you are patching this project, please remove it

Reply:
Looks like you calling are `make` in the `bld.bat`? If so I guess you need it. However, I don't really have much experience with these kind of things on Windows.

Reply:
patch is used only to apply the diff files.

Reply:
So you can remove this dependency, please remove it